### PR TITLE
hw_1_3

### PR DESCRIPTION
**Pull request by IrinaDyakonova**

### CODE REVIEW

File: src/main/java/core/basesyntax/BinaryString.java [hunk 1]
```diff
@@ -7,6 +7,15 @@
...
-        return null;
+        StringBuilder buffer = new StringBuilder();
+        if (value > 0) {
+            while (value != 0) {
+                buffer.append(value % 2);
+                value /= 2;
+            }
+        } else {
+            buffer.append(0);
+        }
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: Do we need else block here ?

Comment (IrinaDyakonova) [reply src/main/java/core/basesyntax/BinaryString.java]: no, if use with do while

File: src/main/java/core/basesyntax/BinaryString.java [hunk 2]
```diff
@@ -7,6 +7,11 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder buffer = new StringBuilder();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BinaryString.java]: let's give better name for buffer, cause there is StringBuffer also so it might be confusing

Comment (IrinaDyakonova) [reply src/main/java/core/basesyntax/BinaryString.java]: done, i renamed variable

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**